ToolBox/CoinApiDataConverter/CoinApiDataReader.cs

```diff
@@ -37,39 +39,24 @@ public class CoinApiDataReader
         /// <param name="tar">The tar input stream</param>
         /// <param name="entry">The tar entry</param>
         /// <returns>A new instance of type <see cref="CoinApiEntryData"/></returns>
-        public CoinApiEntryData GetCoinApiEntryData(TarInputStream tar, TarEntry entry)
+        public CoinApiEntryData GetCoinApiEntryData(FileInfo file, DateTime processingDate, string market)
         {
-            var gzipFileName = entry.Name;
-            Log.Trace($"CoinApiDataReader.ProcessTarEntry(): Processing entry: {gzipFileName}");
+            // crypto/<market>/<date>/<ticktype>-563-BITFINEX_SPOT_BTC_USD.csv.gz
 
-            // datatype-exchange-date-symbol/trades/COINBASE/2019/05/07/27781-COINBASE_SPOT_LTC_BTC.csv.gz
-            var parts = gzipFileName.Split('/');
-            if (parts.Length != 7)
-            {
-                throw new Exception($"CoinApiDataReader.ProcessTarEntry(): Unexpected entry path in tar file: {gzipFileName}");
-            }
+            var tickType = file.FullName.Contains("trade") ? TickType.Trade : TickType.Quote;
 
-            var tickType = parts[1] == "trades" ? TickType.Trade : TickType.Quote;
-            var market = parts[2] == "COINBASE" ? Market.GDAX : parts[2].ToLower();
```

Comment:
Does the new folder structure contain "gdax" instead of "COINBASE"?

Reply:
Good point, and yes, it has.